js/ui/appDisplay.js

```diff
@@ -2478,27 +2478,6 @@ const AppIcon = new Lang.Class({
                     this.activateWindow(null);
                 }));
             }
-
-            if (Shell.AppSystem.get_default().lookup_app('org.gnome.Software.desktop')) {
```

Comment:
How is it broken? I kind of like the feature... (but will not block the PR on it)